Dive Computer/main.c

```diff
@@ -34,41 +77,50 @@ static void TIMER_0_wakeup_task_cb(const struct timer_task *const timer_task)
 	ticks++;
 }
 
-void TIMER_0_example(void)
+void TIMER_0_Init(void)
```

Comment:
Init or init?

Reply:
We need some code convention for those questions.